### ISSUE TITLE
Architecture PPC64LE - Is there a way to provide this architecture too ?

### ISSUE THREAD

**Issue by arthurgermano** [nats-io/nats-docker#117]
Hi there, I apology if it is not the write place to post this message, but I saw in the dockerhub there is no PPC64le architecture. 
So my question is why is not being generated this architecture ?
Please can you start to generate ?

Thanks in advance

**Comment by bruth** [comment]
@arthurgermano In case this is still relevant to you, this arch will be coming for the 2.11.x series of the server.

**Comment by arthurgermano** [comment]
@bruth yes.. it is still very relevant. Many thanks, I am sure that this request will help many developers.. mostly using red hat.. many thanks!
